src/helpers/guilds/scheduledEvents/getScheduledEventUsers.ts

```diff
@@ -25,10 +26,13 @@ export async function getScheduledEventUsers(
 ): Promise<
   Collection<bigint, DiscordenoUser> | Collection<bigint, { user: DiscordenoUser; member: DiscordenoMember }>
 > {
-  // TODO: validate limit
+  if (options?.limit && options.limit > 100) {
```

Comment:
move to plugin

Reply:
Sorry can you elaborate? What do you mean "move to plugin" ? And what action should I take?